src/index.js

```diff
@@ -99,6 +99,7 @@ qtumprocess.on('close', (code) => {
 function exit(signal) {
   console.log(`Received ${signal}, exiting`);
   qtumprocess.kill();
+  process.exit();
```

Comment:
does this need to await for `qtumprocess.kill()` to be done?

Reply:
@dwalintukan .kill() doesnt return `void` instead of `Promise`, so no await

Reply:
👍 :shipit: